includes/class-convertkit-resource-sequences.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * ConvertKit Sequences Resource class.
+ *
+ * @package ConvertKit
+ * @author ConvertKit
+ */
+
+/**
+ * Reads ConvertKit Sequences from the options table, and refreshes
+ * ConvertKit Sequences data stored locally from the API.
+ *
+ * @since   2.5.2
+ */
+class ConvertKit_Resource_Sequences extends ConvertKit_Resource_V4 {
+
+	/**
+	 * Holds the Settings Key that stores site wide ConvertKit settings
+	 *
+	 * @var     string
+	 */
+	public $settings_name = 'convertkit_sequences';
+
+	/**
+	 * The type of resource
+	 *
+	 * @var     string
+	 */
+	public $type = 'sequences';
+
+	/**
+	 * Constructor.
+	 *
+	 * @since   2.5.2
+	 *
+	 * @param   bool|string $context    Context.
+	 */
+	public function __construct( $context = false ) {
```

Comment:
Why is context a boolean here but a string in other places? Why have the default be false here but an empty string other places?

Reply:
Boolean means it won't be included in the User-Agent string for API requests: https://github.com/ConvertKit/convertkit-wordpress-libraries/blob/main/src/class-convertkit-api-v4.php#L1630
A string will be included in the User-Agent string for API requests
Let me know where it's specified as an empty string (I don't see any from searching), and can correct those in a separate PR.